client/src/components/landingpage.js

```diff
@@ -0,0 +1,60 @@
+// landingpage
+
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+
+function LandingPage() {
+
+	const [icon, chooseIcon] = useState('Anything');
+	const [input, updateInput] = useState("");
+	const [placeholder, setPlaceholder] = useState("Enter a City and State or a Zipcode")
+
+	useEffect(() => {
+		if (navigator.geolocation) {
+			navigator.geolocation.getCurrentPosition(displayLocationInfo)
+		}
+	})
+
+	function displayLocationInfo(position) {
+		const lng = position.coords.longitude;
+		const lat = position.coords.latitude;
+		const API_KEY = `${process.env.REACT_APP_GEO_CODER_API_KEY}`;
+		axios.get(`https://api.opencagedata.com/geocode/v1/json?q=${lat}+${lng}&key=${API_KEY}`)
+			.then((response) => {
+				let city = response.data.results[0].components.town
+				let state = response.data.results[0].components.state_code
+				setPlaceholder(`${city}, ${state}`)
+			}, (error) => {
+				console.log(error)
+			})
+	}
+
+	function handleIconClick(e) {
```

Comment:
No arrow function or binding?  Do hooks negate the need for it?  Neat 👍

Reply:
yes hooks negate the need for those because you don't use class components, you use functional components. I'm fairly new to hooks still but I am liking them a lot as I think the code is much cleaner. 

We can also refactor this once we get some sort of state management setup